pkg/testdata/test-shard-identity-bulk.json

```diff
@@ -0,0 +1,29 @@
+{
+  "clusters": [
+    {
+      "name": "cluster1",
+      "locality": "us-west-2",
+      "metadata": {
+        "segment": "seg"
```

Comment:
What is segment?

Reply:
I just added a place holder metadata, currently not used